node/_tools/config.json

```diff
@@ -50,6 +50,7 @@
       "test-fs-mkdir.js",
       "test-fs-open.js",
       "test-fs-opendir.js",
+      "test-fs-promises-file-handle-chmod.js",
       "test-fs-rmdir-recursive.js",
```

Comment:
I'd like to try enabling `test-fs-promises-file-handle-chmod.js`, but running `deno task node:setup` produces a rather large diff (86 files). here i don't know what to do...

![image](https://user-images.githubusercontent.com/40050810/206911783-d5809983-e5db-4ed3-a1c4-cab7895bb03a.png)

Reply:
There seems to be a problem with path resolution on Windows. I'll look into it when I have time.

However, when I tried it locally, I found that this test still fails and I can't enable it. I'm marking this conversation as resolved for now, as the tests don't need to change.